packages/manager/apps/dedicated/client/app/ip/ip/agoraOrder/ip-ip-agoraOrder.constant.js

```diff
@@ -15,7 +15,34 @@ export const PRODUCT_TYPES = {
   },
 };
 
+export const IP_LOCATION_GROUPS = [
+  { labels: ['APAC/CANADA', 'CANADA - ASIA'], countries: ['ca'] },
+  {
+    labels: ['EUROPE'],
+    countries: [
+      'au',
```

Comment:
why sg and au in EUROPE?

Reply:
I've didn't push my updated commit :). Doing